src/Task3/Complex.java

```diff
@@ -0,0 +1,58 @@
+package Task3;
+
+public class Complex {
+    private int real;
+    private int imaginary;
+
+    public Complex() {
+        // No need to do this because default values are set to 0
+        this.real = 0;
+        this.imaginary = 0;
+    }
+
+    public Complex(final int real, final int imaginary) {
+        this.real = real;
+        this.imaginary = imaginary;
+    }
+
+    public int Re() {
+        return this.real;
+    }
+
+    public int Im() {
+        return this.imaginary;
+    }
+
+    public Complex add(Complex z1) {
+        return new Complex(this.real + z1.Re(), this.imaginary + z1.Im());
```

Comment:
Big like for using `Re() `and `Im()`for the parameters and attributes for `this`.

`z1` could have been `final` since I see you are using it, props for that.

Also the variable could have been called `z`.